Ginger/Ginger/ALM/ALMIntegration.cs

```diff
@@ -65,51 +62,106 @@ public void UpdateALMType(eALMType AlmType)
                     }
                     break;
 
-                case eALMType.RQM:
+                case GingerCoreNET.ALMLib.ALMIntegration.eALMType.RQM:
                     AlmCore = new RQMCore();
                     AlmRepo = new RQMRepository();
                     break;
 
-                case eALMType.RALLY:
+                case GingerCoreNET.ALMLib.ALMIntegration.eALMType.RALLY:
                     AlmCore = new RallyCore();
                     AlmRepo = new RallyRepository();
                     break;
-                case eALMType.Jira:
+                case GingerCoreNET.ALMLib.ALMIntegration.eALMType.Jira:
                     AlmCore = new JiraCore();
                     AlmRepo = new JIRA_Repository(AlmCore);
                     break;
+                case GingerCoreNET.ALMLib.ALMIntegration.eALMType.Qtest:
+                    AlmCore = new QtestCore();
+                    AlmRepo = new QtestRepository();
+                    break;
             }
-            if(firstSync)
-                SetALMCoreConfigurations();
+            AlmCore.GetCurrentAlmConfig();
+            SetALMCoreConfigurations(AlmType);
         }
 
-        public void SetALMCoreConfigurations()
+        public void SetALMCoreConfigurations(GingerCoreNET.ALMLib.ALMIntegration.eALMType almType)
         {
+            GingerCoreNET.ALMLib.ALMConfig CurrentAlmConfigurations = GetCurrentAlmConfig(almType);
+
             ALMCore.SolutionFolder = WorkSpace.Instance.Solution.Folder.ToUpper();
-            AlmCore.SetALMConfigurations(WorkSpace.Instance.Solution.ALMServerURL, WorkSpace.Instance.Solution.UseRest, WorkSpace.Instance.UserProfile.ALMUserName, WorkSpace.Instance.UserProfile.ALMPassword, WorkSpace.Instance.Solution.ALMDomain, WorkSpace.Instance.Solution.ALMProject, WorkSpace.Instance.Solution.ALMProjectKey);
-            SyncConfigurations();
+            if(CurrentAlmConfigurations != null)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5395796)